kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -325,7 +327,8 @@ public class BigQuerySinkConfig extends AbstractConfig {
   private static final ConfigDef.Importance MERGE_RECORDS_THRESHOLD_IMPORTANCE = ConfigDef.Importance.LOW;
   private static final String MERGE_RECORDS_THRESHOLD_DOC =
       "How many records to write to an intermediate table before performing a merge flush, if " 
-      + "upsert/delete is enabled. Can be set to -1 to disable record count-based flushing.";
+      + "upsert/delete is enabled. Can be set to -1 to disable record count-based flushing. Either "
+              + MERGE_INTERVAL_MS_CONFIG + " or " + MERGE_RECORDS_THRESHOLD_CONFIG + " must be enabled.";
```

Comment:
Either MERGE_INTERVAL_MS_CONFIG or MERGE_RECORDS_THRESHOLD_CONFIG, or both must be enabled.

Reply:
<img width="1647" alt="Screenshot 2022-12-28 at 3 35 36 PM" src="https://user-images.githubusercontent.com/117655560/209795414-faeceb01-c280-4d54-97cf-9c14bd0b427b.png">